Gems/Atom/Feature/Common/Code/Source/LuxCore/LuxCoreTexture.cpp

```diff
@@ -111,8 +111,12 @@ namespace AZ
                 parentPass->SetSourceTexture(m_texture, RHI::Format::R8G8B8A8_UNORM);
                 break;
             }
-
-            AZ::RPI::RPISystemInterface::Get()->GetDefaultScene()->AddRenderPipeline(m_rtPipeline);
+            
+            const auto mainScene = AZ::RPI::RPISystemInterface::Get()->GetSceneByName(AZ::Name("RPI"));
+            if (mainScene)
+            {
+                mainScene->AddRenderPipeline(m_rtPipeline);
+            }
```

Comment:
Should this be named the LuxCore scene rather than RPI?

Reply:
The LuxCore doesn't create a scene. It uses the scene created by the sample. this is not a best fix. Which I just make it work with ASV. I don't believe the luxcor work with o3de after all...